expected/16.0/type.out

```diff
@@ -997,9 +997,781 @@ SELECT * FROM "type_UUID+" WHERE "u" IS NOT NULL;
+ Foreign Scan on public."type_VARBIT"  (cost=10.00..1824.55 rows=1820 width=53)
+   Output: i, b, (~ (b)::"bit")
+   SQLite query: SELECT `i`, `b` FROM main."type_VARBIT"
+(3 rows)
+
+--Testcase 264:
+SELECT "i", "b", "b" & B'101011' "res" FROM "type_BIT";
+ i |   b    |  res   
+---+--------+--------
+ 6 | 110110 | 100010
+ 7 | 111001 | 101001
+ 8 | 110000 | 100000
+ 9 | 100001 | 100001
+(4 rows)
+
+--Testcase 265:
+SELECT "i", "b", "b" | B'101011' "res" FROM "type_BIT";
+ i |   b    |  res   
+---+--------+--------
+ 6 | 110110 | 111111
+ 7 | 111001 | 111011
+ 8 | 110000 | 111011
+ 9 | 100001 | 101011
+(4 rows)
+
+--Testcase 266:
+SELECT "i", "b", "b" # B'101011' "res" FROM "type_BIT";
+ i |   b    |  res   
+---+--------+--------
+ 6 | 110110 | 011101
+ 7 | 111001 | 010010
+ 8 | 110000 | 011011
+ 9 | 100001 | 001010
+(4 rows)
+
+--Testcase 267:
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" & B'101011') IS NOT NULL;
+ i |   b    
+---+--------
+ 6 | 110110
+ 7 | 111001
+ 8 | 110000
+ 9 | 100001
+(4 rows)
+
+--Testcase 268:
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" | B'101011') IS NOT NULL;
+ i |   b    
+---+--------
+ 6 | 110110
+ 7 | 111001
+ 8 | 110000
+ 9 | 100001
+(4 rows)
+
+--Testcase 269:
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" # B'101011') IS NOT NULL;
+ERROR:  SQL operator is not supported
+HINT:  operator name: #
+--Testcase 270:
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" >> 1) IS NOT NULL;
+ i |   b    
+---+--------
+ 6 | 110110
+ 7 | 111001
+ 8 | 110000
+ 9 | 100001
+(4 rows)
+
+--Testcase 271:
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" << 2) IS NOT NULL;
+ i |   b    
+---+--------
+ 6 | 110110
+ 7 | 111001
+ 8 | 110000
+ 9 | 100001
+(4 rows)
+
+--Testcase 272:
+SELECT "i", "b" FROM "type_BIT" WHERE (~ "b") IS NOT NULL;
+ i |   b    
+---+--------
+ 6 | 110110
+ 7 | 111001
+ 8 | 110000
+ 9 | 100001
+(4 rows)
+
+--Testcase 273:
+EXPLAIN VERBOSE
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" & B'101011') IS NOT NULL;
+                                      QUERY PLAN                                       
+---------------------------------------------------------------------------------------
+ Foreign Scan on public."type_BIT"  (cost=10.00..2203.00 rows=2203 width=13)
+   Output: i, b
+   SQLite query: SELECT `i`, `b` FROM main."type_BIT" WHERE (((`b` & 43) IS NOT NULL))
+(3 rows)
+
+--Testcase 274:
+EXPLAIN VERBOSE
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" | B'101011') IS NOT NULL;
+                                      QUERY PLAN                                       
+---------------------------------------------------------------------------------------
+ Foreign Scan on public."type_BIT"  (cost=10.00..2203.00 rows=2203 width=13)
+   Output: i, b
+   SQLite query: SELECT `i`, `b` FROM main."type_BIT" WHERE (((`b` | 43) IS NOT NULL))
+(3 rows)
+
+--Testcase 275:
+EXPLAIN VERBOSE
+SELECT "i", "b" FROM "type_BIT" WHERE ("b" # B'101011') IS NOT NULL;
```

Comment:
This query should not be error, at least Postgres can process this case without pushing-down

Reply:
Fixed, thanks for discovering corresponding (allow, but not to pushdown operator) code lines for me.

Reply:
Thank for fixing. Please help me to resolve this thread.